sunpy/net/hek/tests/test_hek.py

```diff
@@ -180,7 +180,7 @@ def test_mixed_results_get():
                            attrs.hek.FRM.Name == 'SPoCA')
     assert isinstance(result, hek.hek.HEKTable)
     assert len(result) == 89
-    assert result[0]["SOL_standard"] == 'SOL2013-01-31T20:13:31L253C063'
+    assert result[0]["SOL_standard"].startswith("SOL2013-01-31T20:13:31")
```

Comment:
This protects against the last set of numbers changing.

Reply:
Can you add a comment that says that? Otherwise someone might go back to expecting exact equality

Reply:
Added